jetty-maven-plugin/src/main/java/org/eclipse/jetty/maven/plugin/JettyRunMojo.java

```diff
@@ -144,7 +144,7 @@ private void startScanner()
     {
         if (scan < 0)
         {
-            getLog().info("Redeployment not enabled");
+            getLog().info("default scan=-1:redeploy disabled; scan=0:redeploy manual on ENTER; scan > 0:redeploy automatic");
```

Comment:
@dana-whitelow can you please change this, rather than hardcoding the actual values in the log text, to be something like "Redeployment disabled. See jetty:help for other configuration options."?
Sorry and thanks.